less/v2/chat.less

```diff
@@ -527,9 +525,16 @@
   .targets .target {
     flex-wrap: wrap;
 
+    .subtitle {
+      display: flex;
+      gap: 4px;
+    }
+
     .change-source {
-      width: 16px;
-      padding: 2px;
+      width: 13px;
+      height: 13px;
+      --icon-size: 13px;
```

Comment:
This was as large as I could make it without increasing the height of the overall line, but we could set the `--icon-size: 16px` and keep the button size at 13px which allows the icons to overhang the line without increasing the height of the line itself. I'm not sure myself so it's just 13px for now.